package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "spec-ods",
-  "version": "0.1.0",
+  "version": "0.1.2",
```

Comment:
Where is `0.1.1`?

Reply:
`0.1.1` may have been lost in the testing; however, we can reset the version when we choose for future updates. When using `patch`, `minor`, or `major` in the commits, the version is auto updated on push/PR